### ISSUE TITLE
I get NoSuchMethodError when compiling

### ISSUE THREAD

**Issue by mrt181** [scalapb/ScalaPB#341]
I get a 
`java.lang.NoSuchMethodError: com.trueaccord.scalapb.Scalapb$MessageOptions.getAnnotationsList()Lcom/google/protobuf/ProtocolStringList;`
when trying to run `sbt compile`

## version
`scalaVersion=2.12.3`
`sbt.version=1.0.1`

## operating system
`Microsoft Windows [Version 6.1.7601]`

## build.sbt
```scala
lazy val `xenomorph` = (project in file("."))
  .enablePlugins(GitVersioning)
  .enablePlugins(GitBranchPrompt)
  .enablePlugins(JavaAppPackaging)
  .settings(allSettings)
  .settings(
    organization     := "com.rwe",
    scalaVersion     := "2.12.3",
    scalacOptions    ++= Seq(
      "-unchecked",
      "-deprecation",
      "-language:_",
      "-target:jvm-1.8",
      "-encoding", "UTF-8",
      "-Ywarn-unused-import"
    ),
    javacOptions     ++= Seq(
      "-source", "1.8",
      "-target", "1.8"
    )
  )

val Http4sVersion = "0.15.11a"
lazy val http4sSettings = Seq(
  libraryDependencies ++= Seq(
    "org.http4s"     %% "http4s-blaze-server" % Http4sVersion,
    "org.http4s"     %% "http4s-circe"        % Http4sVersion,
    "org.http4s"     %% "http4s-dsl"          % Http4sVersion,
    "ch.qos.logback"  % "logback-classic"     % "1.2.1"
  )
)

import com.trueaccord.scalapb.compiler.Version.{scalapbVersion, grpcJavaVersion, protobufVersion}
lazy val pbSettings = Seq(
  PB.pythonExe               := "C:\\Python27\\python.exe",
  /*PB.protocVersion           := "-v330",*/
  PB.targets in Compile      := Seq(
    scalapb.gen(flatPackage=true) -> (sourceManaged in Compile).value
  ),
  PB.includePaths in Compile := Seq((sourceDirectory in Compile).value / "proto", file("target/protobuf_external")),
  PB.protoSources in Compile := Seq((sourceDirectory in Compile).value / "proto", file("target/protobuf_external/google/api")),
  libraryDependencies ++= Seq(
    "com.google.api.grpc"     % "googleapis-common-protos" % "0.0.3"          % "protobuf",
    "io.grpc"                 % "grpc-netty"               % grpcJavaVersion,
    "io.grpc"                 % "grpc-services"            % grpcJavaVersion,
    "com.trueaccord.scalapb" %% "scalapb-runtime-grpc"     % scalapbVersion,
    "com.trueaccord.scalapb" %% "scalapb-runtime"          % scalapbVersion   % "protobuf"
  )
)

lazy val gitSettings = Seq(
  git.useGitDescribe := true
)

lazy val configSettings = Seq(
  libraryDependencies ++= Seq(
    "com.github.pureconfig" %% "pureconfig"            % "0.7.2",
    "com.github.pureconfig" %% "pureconfig-enumeratum" % "0.7.2",
    "eu.timepit"            %% "refined"               % "0.8.2",
    "eu.timepit"            %% "refined-pureconfig"    % "0.8.2",
    "com.beachape"          %% "enumeratum"            % "1.5.12"
  )
)

lazy val kafkaSettings = Seq(
  libraryDependencies ++= Seq(
    "org.apache.kafka" % "kafka-streams" % "0.11.0.0"
  )
)

lazy val allSettings = http4sSettings ++
  pbSettings ++
  gitSettings ++
  configSettings ++
  kafkaSettings
```
## plugins.sbt
```scala
addSbtPlugin("com.thesamet"      % "sbt-protoc"            % "0.99.12-rc6")
addSbtPlugin("com.typesafe.sbt"  % "sbt-git"               % "0.9.3")
addSbtPlugin("org.scalastyle"   %% "scalastyle-sbt-plugin" % "1.0.0")
addSbtPlugin("com.typesafe.sbt"  % "sbt-native-packager"   % "1.2.2")

libraryDependencies ++= Seq(
  "com.trueaccord.scalapb"  %% "compilerplugin" % "0.6.3",
  "org.slf4j"                % "slf4j-nop"      % "1.7.21"
)
```
## output
https://gist.github.com/mrt181/54bc478f6e7c96f3516a91320238342c
## expected behavior
generates scala code and compiles
## actual behavior
compilation error

**Comment by xuwei-k** [comment]
duplicate #314

**Comment by mrt181** [comment]
The workaround did not help, only reverting to `0.6.0`

```scala
libraryDependencies ++= Seq(
  "com.trueaccord.scalapb"  %% "compilerplugin" % "0.6.0"
)
```

**Comment by thesamet** [comment]
The problem should be fixed in ScalaPB >= 0.6.4.
